components/smart-home-automation/MQTT-protocol/README.md

```diff
@@ -114,7 +122,7 @@ Sending empty payload to `phoniebox/cmd/help` will be responded by a list of all
 Install missing python packages for MQTT:
 
 ~~~
-sudo pip3 install paho-mqtt
+sudo pip3 install paho-mqtt inotify
```

Comment:
I would suggest to add a `requirements.txt` file to the folder to install the necessary packages. Please see other components how it’s done.

Reply:
Makes sense. Didn't want to do it the "correct way" back then when it was only 1 requirement as the typical phoniebox user is more average than the typical github guys. But as dependencies might increase this makes total sense now.